packages/verification-vue-component/src/style/components/verification/_dropbox.scss

```diff
@@ -52,4 +48,41 @@
             }
         }
     }
+
+    &.dragover {
+        position: absolute;
+        top: 0;
+        height: 100%;
+        width: 100%;
+        background-color: mix($verification-blue, $white, 10%);
+
+        .content {
+            flex-direction: column-reverse;
```

Comment:
Instead of changing the box itself, I would create a separate div, which is overlayed over the content.